selenium_tests/course_info/page_objects/course_info_base_page_object.py

```diff
@@ -1,15 +1,7 @@
-from selenium.common.exceptions import NoSuchElementException
 from selenium_common.base_page_object import BasePageObject
 
 
 class CourseInfoBasePageObject(BasePageObject):
-    """
-    This is the base page object class that all Course Info pages can inherit from
-    Locators and Services would be common to all pages on  this tool
-    """
     def __init__(self, driver):
         super(CourseInfoBasePageObject, self).__init__(driver)
-        try:
-            self._driver.switch_to.frame(self._driver.find_element_by_id("tool_content"))
-        except NoSuchElementException:
-            pass
+        self.focus_on_tool_frame()
```

Comment:
Re: moving this into selenium common, agreed. On a related note, this is the first time we've cached the POs as TestCase class objects, instead of instantiating them for each new run, so the focus_on_tool_frame isn't as helpful -- since the POs are only instantiated once, in some ways we have to be more careful we're not leaving the tool frame in any tests, or that if we do we refocus at the beginning of the tests. So I think it belongs in selenium common (eventually), but the class-level caching might not work for all test suites...